app/domain/authentication/authn_jwt/identity_providers/identity_from_decoded_token_provider.rb

```diff
@@ -9,108 +9,78 @@ module IdentityProviders
           fetch_identity_path: Authentication::AuthnJwt::IdentityProviders::FetchIdentityPath.new,
           fetch_authenticator_secrets: Authentication::Util::FetchAuthenticatorSecrets.new,
           check_authenticator_secret_exists: Authentication::Util::CheckAuthenticatorSecretExists.new,
-          add_prefix_to_identity: Authentication::AuthnJwt::IdentityProviders::AddPrefixToIdentity.new,
           logger: Rails.logger
         },
         inputs: %i[jwt_authenticator_input]
       ) do
-        extend(Forwardable)
-        def_delegators(:@jwt_authenticator_input, :service_id, :authenticator_name, :account)
```

Comment:
Why removing the deligator?

Reply:
Good question, I meant to address this in my writeup.

Over time, I've come to the conclusion that using `Forwardable` is usually not worth it.  I have a  few reasons for this:

1. The "benefit" is minimal.  Ostensibly, the benefit is to make the calling site cleaner.  You can just write `my_method` instead of `@my_object.my_method`.  This is a minor benefit, unless you are calling `my_method` many times, which we usually aren't.
2. It adds a layer of abstraction for the reader.  If you've jumped into the code from somewhere else, you'll have to do an extra lookup to determine how the method is defined.  What's worse, you can't do this jump automatically, because....
3. It breaks tooling.  You can't use your IDE's "Jump to definition..." feature for methods defined this way.  Granted, one of the downsides of the CommandClass is that the same is true of jumping to dependencies, but there's no need to compound the problem.  Also, methods defined in this way don't show up in the "structure" view, which is a really nice way to understand classes:

<img width="1650" alt="image" src="https://user-images.githubusercontent.com/699684/134246300-f128bbe5-2c1b-4c67-b304-072ee7ad2bee.png">